src/lesson-2-1-demo/ClassComponents.jsx

```diff
@@ -1,96 +1,62 @@
 import { faker } from "@faker-js/faker";
-import { createContext, Component } from "react";
+import {
+  createContext,
+  Component,
+  useContext,
+  useState,
+  useEffect,
+} from "react";
 
-export class HelloWorld extends Component {
-  render() {
-    return <p>Hello, World!</p>;
-  }
-}
-
-export class Hello extends Component {
-  render() {
-    const { name } = this.props;
-    return <p>Hello, {name}!</p>;
-  }
-}
+export const HelloWorld = () => <p>Hello, World!</p>;
```

Comment:
everything from the `render()` function is moved to the output of a functional component